packages/core/package.json

```diff
@@ -211,13 +99,14 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "jest",
-    "build": "rm -rf ./dist && NODE_OPTIONS=\"--max-old-space-size=12288\" bunchee",
+    "build": "rm -rf ./dist && pnpm run build:esm && pnpm run build:cjs && pnpm run build:type",
```

Comment:
as you're using vitest now, can we still trigger it from the core package?

Reply:
why did you change to vitest?

Reply:
Jest doesn't support ESM and needs lots of work on configuration. ALSO not support typescript natively